packages/vue-app/template/client.js

```diff
@@ -827,7 +827,7 @@ async function mountApp (__app) {
   // Load page chunk
   if (!NUXT.data && NUXT.serverRendered) {
     try {
-      const payload = await _app.fetchPayload(_app.context.route.path)
+      const payload = await _app.fetchPayload(NUXT.routePath || _app.context.route.path)
```

Comment:
We may add `NUXT.routePath` to [spa renderer context](https://github.com/nuxt/nuxt.js/blob/f585e46829d7e704b98ca0492350e03d9fde1ce5/packages/vue-renderer/src/renderers/spa.js#L157) and always use it instead of `_app.context.route.path` then we may be able to remove `context.route.path` (just i think we may to do sanetization similar to `vue-router` on baseURL and trailing slash (for `/`)

Reply:
Well here we are in the condition if `NUXT.serverRendered` so it won't change to have it in the SPA renderer right?